flake.nix

```diff
@@ -1,12 +1,35 @@
 {
   inputs = {
     nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
-    flake-utils.url = "github:numtide/flake-utils";
+    std = {
+      url = "github:divnix/std";
```

Comment:
This is the first time I've really looked at std this close so I'm not really comfortable but it doesn't look too bad